ui/helpers/utils/util.js

```diff
@@ -408,3 +408,18 @@ export function getURLHost(url) {
 export function getURLHostName(url) {
   return getURL(url)?.hostname || '';
 }
+
+// Once we reach this threshold, we switch to higher unit
+const MINUTE_CUTOFF = 90 * 60;
+const SECOND_CUTOFF = 90;
+
+export const toHumanReadableTime = (milliseconds = 1, t) => {
```

Comment:
What do you think about adding a test for this so it's easier to understand the behavior of this function better? Seems like there's some nuances here that aren't fully readable from the code.

Reply:
Also, based on how you use this function, it seems that you always pass `milliseconds` — what do you think about not making it optional? Or if you really want it to be optional, perhaps have it be the second argument? It feels like it's more natural for optional arguments to appear after required ones.

Reply:
testing it is not so straightforward as it needs localisation support which is available to only react components.

about 1st optional argument, good point let me fix